ansible/playbooks/install-explorer.yml

```diff
@@ -1,9 +1,17 @@
 ---
 - hosts: '{{ inventory }}'
   remote_user: '{{ user }}'
-  become: true
-  become_user: root
   gather_facts: yes
   roles:
-  - hmyserver
-  - hmyrepo
+    - nodejs
+    - certbot_auto
+    - explorer_fe
+    - explorer_be
+  vars:
+    nodejs_version: '10.18.0'
+    domain: 'explorer.testnet.harmony.one'
+    firebase_domain: 'harmony.testnet.firebaseio.com'
```

Comment:
what is this? are you referring to the firebase database url?
in that case, it should be databaseURL: `https://<DATABASE_NAME>.firebaseio.com`
so `https://harmony-explorer-backend-b073a.firebaseio.com`

Reply:
ummm... it is a fake URL... we should set it as param when run the playbook